### PR TITLE
Feat/merge back

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

File: src/renderer/selfkey-id/main/components/selfkey-id-applications.jsx [hunk 1]
```diff
@@ -261,9 +271,254 @@ const getRpInfo = (rpName, field) => {
 const MARKETPLACE_ROOT_PATH = '/main/marketplace-categories';
 
 class SelfkeyIdApplicationsComponent extends Component {
```
Comment (sk91) [inline src/renderer/selfkey-id/main/components/selfkey-id-applications.jsx]: @rodrigopavezi are you sure this is right? I wouldnt expect ui to be only on release branch maybe it is overriding some changes we did in dev branch?

File: src/renderer/selfkey-id/main/components/selfkey-id.jsx [hunk 2]
```diff
@@ -1,302 +1,38 @@
 import React, { Component } from 'react';
-import CircularProgress from '@material-ui/core/CircularProgress';
-import { Grid, Typography, Tabs, Tab, Button, withStyles } from '@material-ui/core';
-import { connect } from 'react-redux';
```
Comment (sk91) [inline src/renderer/selfkey-id/main/components/selfkey-id.jsx]: Same here about ui

File: src/renderer/selfkey-id/main/components/selfkey-id-applications.jsx (hunk 1 above)
Comment (rodrigopavezi) [reply src/renderer/selfkey-id/main/components/selfkey-id-applications.jsx]: @sk91 that is correct. It was only changed on the release branch and there was no merging issue when I merge dev into this branch.

File: src/renderer/selfkey-id/main/components/selfkey-id.jsx (hunk 2 above)
Comment (rodrigopavezi) [reply src/renderer/selfkey-id/main/components/selfkey-id.jsx]: Same here too

**Pull request merged**